arch/arm/src/phy62xx/uart.c

```diff
@@ -62,6 +68,9 @@ typedef struct _uart_Context
   uint8_t       tx_state;
   uart_Tx_Buf_t tx_buf;
   uart_Cfg_t    cfg;
+  uint8_t fifo_buf_store[UART_RX_BUFFER_SIZE];
+  int buf_head;
+  int buf_tail;
 } uart_Ctx_t;
 
 static uart_Ctx_t m_uartCtx[2] =
```

Comment:
Why reserve so much memory for devices with limited available resources? Is it necessary to put `UART_RX_BUFFER_SIZE` into kconfig?

Reply:
ok, i will reduce the RX_BUFF_SIZE and self check first.